client/app/widgets/QuickCreate/index.jsx

```diff
@@ -4,10 +4,9 @@ import { Modal } from '../../components/Modal';
 import { Input } from '../../components/Input';
 import type { Checkbox } from '../../components/Input/utils';
 import { Utils } from '../../utils';
-// eslint-disable-next-line import/no-cycle
 import css from './QuickCreate.scss';
 // eslint-disable-next-line import/no-cycle
-import { DynamicForm } from '../../components/Form/DynamicForm';
+import { DynamicForm} from '../../components/Form/DynamicForm';
```

Comment:
Looks like you'll want to do `import { DynamicForm }` instead of `import { DynamicForm}` since this will raise a linting error i.e.

```
/home/circleci/ifmeorg/ifme/client/app/widgets/QuickCreate/index.jsx
  9:21  error  A space is required before '}'  object-curly-spacing

✖ 1 problem (1 error, 0 warnings)
  1 error and 0 warnings potentially fixable with the `--fix` option.
```

Reply:
If you run `yarn lint` in the client folder locally, it will fix things for you automatically!